SEED-platform/seed
# fix: correctly check if import file has meters tab

#### Any background context you want to provide?

#### What's this PR do?
Fixes the bug in issue

#### How should this be manually tested?
- upload a BSync file, it should not prompt you to import meters
- import a spreadsheet with meters. it should prompt you to import meters

#### What are the relevant tickets?
#2810 

#### Screenshots (if appropriate)
